cordformation/src/main/kotlin/net/corda/plugins/Baseform.kt

```diff
@@ -66,6 +68,26 @@ open class Baseform : DefaultTask() {
     @Input
     var nodeDefaults: Closure<in Node>? = null
 
+    /**
+     * Optional parameters for ant keyGen and signJar tasks to sing Cordapps
+     */
+    @Input
+    var signCordapps: MutableMap<String,String> = mutableMapOf()
```

Comment:
This API should really be strongly typed. Thankfully the SignJar task is on the classpath for this project already so you can instead use;

    fun signCordapp(signJar: Closure<in SignJar>) {
        signJarTask = project.configure(SignJar(), signJar) as SignJar
    }

Below you have defaults so you can instead create a SignJar() with default values and pass _that_ into the project.configure().

Then the output of project.configure will be the callable ant task. This removes the stringly typing we have here.

The API should also be;

signing { // This uses a new Signing class
    enabled: boolean = false
    jarSign: Closure<JarSign>
}

This ensures we stay strongly typed with the actual task itself without duplicating it in a wrapper class (we always can later if we need to) and also means we could add more kinds of signing if we ever need to or other meta-information (ie; enabled, selective signing of Cordapps, etc).